src/lib/structures/expr.ml

```diff
@@ -32,6 +32,13 @@ module SSet = Sy.Set
 
 (** Data structures *)
 
+let compare_list = List.compare
+let compare_bool = Bool.compare
+let compare_int = Int.compare
+let equal_list = List.equal
+let equal_bool = Bool.equal
+let equal_int = Int.equal
```

Comment:
these should not be necessary, ppx_compare should be smart enough to understand that `list` is actually `List.t`, no ?

Reply:
If I put `List.t` in the record/adt, it works of course but that's ugly.